index.js

```diff
@@ -235,33 +234,35 @@ function schemaMiddleware (schema) {
 }
 
 function cors (opts) {
-  var _cors = corsify(opts)
+  opts = opts || {}
+  var headers = opts.headers || ['Content-Type', 'Accept', 'X-Requested-With']
+  var methods = opts.methods || ['PUT', 'POST', 'DELETE', 'GET', 'OPTIONS']
```

Comment:
I see your point about concating the array first; I didn't know about the difference compression makes in requests tbh. So let's do that after explicitly setting an array there. 

Single value is totally fine in this case since it just sets the one (already tested :D)